src/amo/reducers/viewContext.js

```diff
@@ -10,12 +10,16 @@ import {
 
 
 export type ViewContextType = {|
-  context: ADDON_TYPE_EXTENSION | ADDON_TYPE_THEME | VIEW_CONTEXT_EXPLORE |
-    VIEW_CONTEXT_HOME | VIEW_CONTEXT_LANGUAGE_TOOLS,
+  context:
+    | typeof ADDON_TYPE_EXTENSION
```

Comment:
Why did we have to add `typeof` to each of these? I assume Flow was fine with it before.

Reply:
That does not work anymore. Flow complains that we are using a string constant and that it should be prefixed with `typeof`. Ideally, we should add types to each constant but then new errors pop up 😞